support/utils/configs.go

```diff
@@ -27,18 +27,23 @@ func StructString(s interface{}, transforms map[string]func(interface{}) interfa
 	var buf bytes.Buffer
 	numFields := reflect.TypeOf(s).NumField()
 	for i := 0; i < numFields; i++ {
-		fieldName := reflect.TypeOf(s).Field(i).Name
+		field := reflect.TypeOf(s).Field(i)
+		fieldName := field.Name
+		fieldDisplayName := field.Tag.Get("toml")
+		if fieldDisplayName == "" {
+			fieldDisplayName = fieldName
+		}
 
 		// set the transformation function
 		transformFn := passthrough
-		if fn, ok := transforms[fieldName]; ok {
+		if fn, ok := transforms[fieldDisplayName]; ok {
```

Comment:
using `fieldName` here requires the `transforms` map to use the _variableName_, whereas using `fieldDisplayName` requires the map to use the _CONFIG_NAME_.
see related comment in `terminator/config.go`.